README/ReleaseNotes/v616/index.md

```diff
@@ -49,6 +49,17 @@ They were deprecated before, or never ported from configure, make to CMake.
 
 `. bin/thisroot.fish` sets up the needed ROOT environment variables for one of the ROOT team's favorite shells, the [fish shell](https://fishshell.com/).
 
+### Change of setting the compression algorithm in `rootrc`
+
+The previous setting called `ROOT.ZipMode` is now unused and ignored.
```

Comment:
This is suboptimal as it means that **silently** the compression algorithm choice is ignored (users may have used values other than the one advertised in the comments).

It would be better to continue to use it when the new flag is not set or at least to warn it is being ignored when it is not the default value (nor a request from the broken zlib version)

Reply:
Thanks. Does this address it? https://github.com/root-project/root/pull/2363